vms/proposervm/block/block.go

```diff
@@ -73,32 +80,82 @@ func (b *statelessBlock) Block() []byte {
 	return b.StatelessBlock.Block
 }
 
+func (b *statelessBlock) VRFSig() []byte {
+	return b.StatelessBlock.VRFSig
+}
+
+func (b *statelessBlock) PChainHeight() uint64 {
+	return b.StatelessBlock.PChainHeight
+}
+
+func (b *statelessBlock) Timestamp() time.Time {
+	return b.timestamp
+}
+
+func (b *statelessBlock) Proposer() ids.NodeID {
```

Comment:
Nit: we could reduce the diff in this PR by keeping these methods in their original place but idc if they move

Reply:
done.